spec/models/review_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe Review, type: :model do
+
+  it { should have_valid(:rating).when(1)}
+  it { should_not have_valid(:rating).when(10)}
+  it { should_not have_valid(:rating).when(nil, "")}
+  it { should have_valid(:body).when("grayworm")}
+  it { should_not have_valid(:body).when(nil, "")}
+end
```

Comment:
it would be great if we had association tests as well here

Reply:
I'm trying to get the syntax for this, and I'm leaning towards wondering if shoulda needs to be reconfigured. I am trying this in the Stop model test  ` it { should have_many(:reviews) }` and it seems like shoulda isn't even understanding the test. Am I doing something wrong?
This is what rspec gives me 
`Failures:

  1) Stop should have many :reviews
     Failure/Error: it { should have_many(:reviews) }
       expected #<Stop:0x00007ffd919e9620> to respond to `has_many?`
     # ./spec/models/stop_spec.rb:5:in `block (2 levels) in <top (required)>'
`